feat: FAQ - Add LUKSO Chapter (#572)
lukso-network/docs: docs/faq/lukso/general-information.md

---
sidebar_label: 'General Information'
sidebar_position: 1
---

# General Information

### What is the LUKSO Network?

LUKSO is a Layer-1 blockchain network dedicated to existing and coming digital lifestyles and creative use cases. It is built on the EVM framework and compatible with all other platforms or protocols built on Ethereum.

### Will LUKSO be the main platform to integrate LSPs?

> The LUKSO blockchain will be the first mover for LUKSO Standard Proposals. In conjunction with the surrounding ecosystem, Universal Profiles greatly enhance the interaction and convenience of managing LSPs. This advantage stems from the **seamless integration of Universal Profiles**. With a dedicated network like LUKSO, we can implement well-thought-out, upgradable standards directly from the start. This clean-cut approach allows us to bypass the challenges of retrofitting standards into an existing system.

Reviewer: Maybe you can add a link to the LSP introduction page?
https://docs.lukso.tech/standards/introduction

I think we should mention that the LSPs can be ported and implemented to any other EVM based blockchain.

```suggestion
The LUKSO blockchain is the first mover to integrate the LUKSO Standard Proposals (LSPs). In conjunction with the surrounding ecosystem, Universal Profiles greatly enhance the interaction and convenience of using the blockchain and dApps. This advantage stems from the **seamless integration of Universal Profiles**. With a dedicated network like LUKSO, we can implement well-thought-out, upgradable standards directly from the start. This clean-cut approach allows us to bypass the challenges of retrofitting standards into an existing system.
```

> ### Who defined the LUKSO token supply?

Reviewer 2: maybe you can add a link to this article which also explains this: https://medium.com/lukso/its-happening-the-genesis-validators-are-coming-ce5e07935df6

The genesis validators of LUKSO collectively determined the token distribution of 42M LYX by voting for it while depositing LYXe, after 3 different options were given by LUKSO. After the [Genesis Validator Contract](https://etherscan.io/address/0x42000421dd80D1e90E56E87e6eE18D7770b9F8cC#code) was closed, the network started with the related genesis files.

### Why did LUKSO launch as Layer-1 Blockchain?

LUKSO is an L1 blockchain designed to provide **distinct security, interoperability, and adaptability advantages**, creating a network effect that supports a newly independent economy without compromising its key aspects.

> LUKSO is dedicated to constructing standards that **significantly enhance the user experience of blockchain** technology. However, the full potential of Universal Profiles can't be realized if only a fraction of the network is onboarded through it. Interactions between profiles bring substantial value, and making Universal Profiles the initial accounts through which users are joining guarantees easy usability and upgradability for everyone. Creating a new network offers a space to **evolve and refine** these new concepts **without the hindrance of constraints** from current landscapes and without the obligation of being fully backward compatible.

Author: Great upgrade! thanks 😘

Despite L2 solutions offering lower costs, they often suffer from interoperability issues, governance concerns, and the need for rolling back data to the main network, **risking the siloing of assets and profiles**. Depending on the rollup type, different sacrifices like calculation, smart contract execution, or finality time have to be made on top.

Rollups, while providing scalability, present their challenges. They are only as secure as the speed at which assets can be exited to the main chain. Moreover, **not every protocol or activity** implemented on the rollup **can be exited**, as not everything is an asset. This limits their applicability and functionality for new account systems.

While standalone applications might not see the necessity of having their own Layer-1 network, LUKSO provides a **distinct realm for a new** core blockchain **account ecosystem**, necessarily needing the robust and secure foundation to utilize its full potential.

### What's the token distribution of LUKSO?

> Please look at our official [Network Migration](https://etherscan.io/address/0x42000421dd80D1e90E56E87e6eE18D7770b9F8cC#code) article to see the distribution amounts and listing of the final supply. Previous distributions have been outlaid in our [Genesis Validator](https://medium.com/lukso/its-happening-the-genesis-validators-are-coming-ce5e07935df6) announcement.

Reviewer 2: is it the correct link? I would expect a link to an article, maybe you wanted to put this?

https://medium.com/lukso/the-lyxe-migration-process-374053e5ddf5

### What's the difference between LYXt, LYXe, and LYX?

- **LYXt** is the currency for developers used on our official LUKSO Testnet without monetary value. LYXt can be required for free using the [Testnet Faucet](https://faucet.testnet.lukso.network/). The Testnet is maintained by a set of whitepisted node operators to ensure stability.

> - **LYXe** is the representation of LYX on the Ethereum Blockchain, used to start off the LUKSO Mainnet in a distributed way. It was created in 2020 during the [rICO](https://medium.com/lukso/re-launching-the-reversible-ico-5289989ce7ed) of LUKSO and started as main token before the mainnet was released.

Reviewer 2: Maybe you can put a link to:

https://etherscan.io/token/0xA8b919680258d369114910511cc87595aec0be6D

Reviewer: Maybe you can also word it as _"it's the LUKSO token on Ethereum"_
